doc/sphinx/source/projects/ad355xr_iio/ad355xr_iio.rst

```diff
@@ -0,0 +1,98 @@
+AD355XR IIO Application
```

Comment:
There is no mention about the STM32 support in the FW. Is there a reason why it is not mentioned?

Reply:
i referred to other documents. In Name we are not mentioning MBED/STM

Reply:
Not in the name- I meant in the documentation itself